### PR TITLE
Datastore Feature

### PR DESCRIPTION
**Pull request by mariah-crawford**
Add doPost function that gets the mood scale input from the form and stores it as a Journal entity in Datastore

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,19 +14,42 @@
...
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (gracieorozco) [inline src/main/java/com/google/sps/servlets/DataServlet.java:27]: The description for the servlet file should be updated to reflect what it's functionality is.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@ <h1>Our webapp</h1>
       <p>This is our webapp.</p>
       <p>Click here to write a journal entry:</p>
 
-      <form action= "/my-data-url" id="my-form" method="POST" enctype="multipart/form-data">
+      <form action= "/my-data-url" method="POST" >
```
Comment (gracieorozco) [inline src/main/webapp/index.html:16]: Removing the space after the "action=" and before the last ">" would help format the form layout better. Below on line 20, the extra space before the "/>" would be helpful too.

**Review by gracieorozco — APPROVED** [review]
Great work on implementing the DataStore. The forms and servlet are working as expected as I have tested this branch. There are a few minor edits to change, but they are mainly format and layout changes.

File: src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (mariah-crawford) [reply src/main/java/com/google/sps/servlets/DataServlet.java:27]: Done :)

File: src/main/webapp/index.html (hunk 2 above)
Comment (mariah-crawford) [reply src/main/webapp/index.html:16]: Done :)

**Pull request merged**